src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al

```diff
@@ -54,7 +53,7 @@ page 7775 "Copilot AI Capabilities"
             {
                 ShowCaption = false;
                 InstructionalText = 'Generative AI capabilities are deactivated because Azure OpenAI Service is not available in your region. By allowing data movement, you agree to data being stored and processed by the Azure OpenAI Service outside of your environment''s geographic region or compliance boundary.';
-                Visible = not InGeo;
+                Visible = not WithinGeo;
 
                 field(DataMovement; AllowDataMovement)
```

Comment:
I don't think we need to have a separate group, it still applies. As it's the compliance boundary that we're checking. 
For Europe, if a country returns `WithinGeo=true` that would mean the resources we need are within the compliance boundary. 
Unless we're saying for all European countries, we'll always show this separate text + the toggle. Mike is OOF, so this decision can't be made atm. I'll leave this PR open till he's back for us to discuss. I'm on the side that this is fine, but he may propose otherwise.